nixos/hosts/lab/compose.lab.yml

```diff
@@ -2,7 +2,7 @@
 #
 # SPDX-License-Identifier: MPL-2.0
 
-services:
-  rathole:
-    links:
-      - actual
+networks:
+  internet:
+  internal-proxy:
+    internal: true
```

Comment:
this means that there is no internet access for the container too, so we need to be careful